src/GraphQL/Types/GraphTypesLookup.cs

```diff
@@ -107,6 +107,20 @@ public void AddType(GraphType type, TypeCollectionContext context)
                 {
                     AddType(context.ResolveType(field.Type), context);
                 }
+
+                if (field.Arguments == null)
+                {
+                    return;
```

Comment:
I think this return could cause issues with interfaces not getting registered (the `ObjectGraphType` check after this one).